setup.py

```diff
@@ -1,11 +1,16 @@
 import os.path
 from setuptools import setup
 
-with open(os.path.join(os.path.dirname(__file__), "README.md")) as f:
+with open(
+    os.path.join(os.path.dirname(__file__), "README.md"), encoding="utf-8"),
```

Comment:
I somehow missed it and the project doesn't build test via the CI. Fixed in <https://github.com/akamhy/waybackpy/pull/116>.

Reply:
The issue is with the "," at the end of line 5.